ipath/ipath_time.c

```diff
@@ -278,8 +289,10 @@ ipath_timebase_from_cpuinfo(uint32_t old_pico_per_cycle)
     }
 #endif
 
+    max_bet_new_old_pico = max(new_pico_per_cycle, old_pico_per_cycle);
+    min_bet_new_old_pico = min(new_pico_per_cycle, old_pico_per_cycle);
     /* If there's no change (within a small range), just return the old one */
-    if (abs(new_pico_per_cycle - old_pico_per_cycle) < 5)
+    if ((max_bet_new_old_pico - min_bet_new_old_pico) < 5)
```

Comment:
@hestela As far as I can see you are contributing to the psm. Could you take a look or advice some expert of the PSM who can help me, please? Thanks in advance and sorry for disturbing you.